hydra/_internal/config_loader_impl.py

```diff
@@ -266,6 +266,13 @@ def _load_configuration_impl(
             for key in cfg.hydra.job.env_copy:
                 cfg.hydra.job.env_set[key] = os.environ[key]
 
+        for source in caching_repo.get_sources():
+            if not source.available():
+                warnings.warn(
+                    category=UserWarning,
+                    message=f"provider={source.provider}, path={source.path} is not available.",
+                )
+
```

Comment:
This validation is better done right after the sources settles down.
Otherwise you will potentially get errors loading configs that are not found because the searchpath is incorrect - before this warning is issued.